bin/dephpugger

```diff
@@ -1,89 +1,32 @@
 #!/usr/bin/env php
 <?php
 
-$autoloads = [
-    dirname(dirname(dirname(dirname(__DIR__)))) . DIRECTORY_SEPARATOR,
-    
+$availableAutoloadPaths = [
+    __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'autoload.php',
```

Comment:
If I install in global mode, isn't possible to find the composer.
Need search in 4 paths before too.

Only add this line:

```
    __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'autoload.php',
```

When run dephpugger:
```
/Users/username/.composer/vendor/tacnoman/dephpugger/bin
```

4 paths before.

```
/Users/usernname/.composer/vendor/tacnoman/dephpugger/bin/../../../autoload.php
```

Equals to:
```
/Users/usernname/.composer/vendor/autoload.php
```